repository/dev/javamodules/jpms-catalog/1.0.10/jpms-catalog-1.0.10.toml

```diff
@@ -0,0 +1,50 @@
+#
+# This file has been generated by Gradle and is intended to be consumed by Gradle
+#
+[metadata]
+format.version = "1.1"
+
+[versions]
+attic = "1.0.10"
+checker = "3.43.0-SNAPSHOT"
+errorprone = "2.26.1-jpms"
+geantyref = "1.3.15-jpms"
+guava = "33.1.0-jre-jpms"
+j2objc = "3.0.0"
+kotlinx-collections-immutable = "0.4.1-jpms"
+maven-resolver = "2.10.0-alpha-10-jpms-b3"
+protobuf = "4.26.0-jpms"
+reactivestreams = "1.0.4-jpms"
```

Comment:
needs updating:

- drop outdated libs like `checker` and `errorprone`, or update to latest JPMS-supported versions from central
- fixup version for `guava`
- fixup version for `reactivestreams` (it's at `1.0.5` now)